app/src/ui/history/commit-list.tsx

```diff
@@ -194,9 +218,10 @@ export class CommitList extends React.Component<ICommitListProps, {}> {
         <List
           rowCount={this.props.commitSHAs.length}
           rowHeight={RowHeight}
-          selectedRows={[this.rowForSHA(this.props.selectedSHA)]}
+          selectedRows={this.props.selectedSHAs.map(sha => this.rowForSHA(sha))}
```

Comment:
Not saying you have to do this, but you could, right? Again, nothing to do in this PR, just curious to know what is your opinion 🤣 
```suggestion
          selectedRows={this.props.selectedSHAs.map(this.rowForSHA)}
```

Reply:
It doesn't work.. I tried that first. 😄  Scoping of "this"
![image](https://user-images.githubusercontent.com/75402236/110362169-f13aeb80-800e-11eb-8306-045426ec8d0c.png)